File: maven-model-reader-plugin/src/main/java/ru/rzn/gmyasoedov/model/reader/plugins/GroovyMavenPlusPlugin.java
package ru.rzn.gmyasoedov.model.reader.plugins;

import org.apache.maven.model.Plugin;
import org.apache.maven.project.MavenProject;

public class GroovyMavenPlusPlugin implements PluginProcessor{
    @Override
    public String groupId() {
        return "org.codehaus.gmavenplus";
    }

    @Override
    public String artifactId() {
        return "gmavenplus-plugin";
    }

    @Override
    public void process(MavenProject project, Plugin plugin) {
        AbstractGroovyPluginProcessor.process(project, plugin);
    }
}

File: gmaven/src/main/kotlin/ru/rzn/gmyasoedov/gmaven/extensionpoints/plugin/GroovyMavenPlusPlugin.kt
package ru.rzn.gmyasoedov.gmaven.extensionpoints.plugin

class GroovyMavenPlusPlugin : MavenFullImportPlugin {
    override fun getGroupId() = "org.codehaus.gmavenplus"

    override fun getArtifactId() = "gmavenplus-plugin"
}